test/parallel/test-timers-promisified.js

```diff
@@ -392,18 +392,23 @@ process.on('multipleResolves', common.mustNotCall());
   // Check that the timing is correct
   let pre = false;
   let post = false;
-  setPromiseTimeout(1).then(() => pre = true);
-  const iterable = setInterval(() => {}, 2);
-  const iterator = iterable[Symbol.asyncIterator]();
-
-  iterator.next().then(common.mustCall(() => {
-    assert.ok(pre, 'interval ran too early');
-    assert.ok(!post, 'interval ran too late');
-    return iterator.next();
-  })).then(common.mustCall(() => {
-    assert.ok(post, 'second interval ran too early');
-    return iterator.return();
-  }));
 
-  setPromiseTimeout(3).then(() => post = true);
+  const time_unit = common.platformTimeout(50);
```

Comment:
Optional nit: Since the problem we've been seeing is with the test being flaky on a fast machine, I wonder if `common.platformTimeout()` isn't needed here. It's usually for giving slow machines more time, but the problem we're seeing right now is the other way around--our LinuxONE host is too efficient.